packages/formation/sass/modules/_m-loading-indicator.scss

```diff
@@ -15,3 +27,12 @@
 .async-loader {
   padding-bottom: 1em;
 }
+
+@keyframes spin {
```

Comment:
Thoughts on moving this to a `_keyframes.scss` file?

Reply:
Hm, I see it either way, but for consistency across the design system it's probably better in `_keyframes.scss`.

Reply:
Hm, actually, I thought we already had a keyframes file, but since we don't it might be better to do this separately. What do you think?